templates/base_index.html

```diff
@@ -1375,17 +1375,40 @@ <h2>
     if (window.ActiveXObject) {
       xhr = new ActiveXObject('Microsoft.XMLHTTP');
     }
+    var fetchUserCountry = new XMLHttpRequest();
+    if (window.ActiveXObject) {
+      fetchUserCountry = new ActiveXObject('Microsoft.XMLHTTP');
+    }
 
     xhr.onreadystatechange = function() {
       if (xhr.readyState == XMLHttpRequest.DONE) {
         if (xhr.status === 200) {
           var takeovers = JSON.parse(xhr.responseText);
 
           // Get the selected takeovers based on the primary language
-          var selectedTakeovers = takeovers.filter(function(item) {
-            return item.lang === primaryParentLanguage || item.lang === ""
-          })
-
+          
+          if (fetchUserCountry.readyState == XMLHttpRequest.DONE && fetchUserCountry.status == 200) {
+            var userCountry = JSON.parse(fetchUserCountry.responseText).country_code;
+            var selectedTakeovers = takeovers.filter(function(item) {
+              if (item.target_country && item.target_country == userCountry) {
+                return true
+              } else if (item.lang === primaryParentLanguage || item.lang === "") {
+                return true
+              } else {
+                return false
+              }
+            })
+
+          } else {
+            var selectedTakeovers = takeovers.filter(function(item) {
+            if (item.lang === primaryParentLanguage || item.lang === "") {
+                return true
+              } else {
+                return false
+              }
+            })
+          }
```

Comment:
Do you use prettier? Your semi-colons are missing

Reply:
Prettier doesn't work well when I use it mixed with Jinja and HTML